### DISCUSSION TITLE
Can the following parameters be used to connect to the storage of the S3 interface protocol?

### DISCUSSION THREAD

**Discussion by a28124791** [milvus-io/milvus#36716] [Q&A and General discussion]
Can the following parameters be used to connect to the storage of the S3 interface protocol?
使用下面的参数是否可以连接s3接口协议的存储？

## rootcoord
docker run -d -it \
  --name milvus_rootcoord-up2 \
  -p 53100:53100 \
  -e "ETCD_ENDPOINTS=192.168.15.10:12379" \
  -e "ETCD_ROOTPATH=gbotby" \
  -e "ETCD_METASUBPATH=meta" \
  -e "ETCD_KVSUBPATH=kv" \
  **-e "MINIO_ADDRESS=192.168.15.10" \
  -e "MINIO_PORT=9878" \
  -e "MINIO_ACCESS_KEY_ID=werwerwerwerwrwrw" \
  -e "MINIO_SECRET_ACCESS_KEY=ererererererererr" \**
  -e "PULSAR_ADDRESS=pulsar://192.168.15.10:6650" \
  -e "ROOT_COORD_ADDRESS=192.168.15.10:53100" \
  -e "ROOT_COORD_ENABLE_ACTIVE_STANDBY=true" \
milvusdb/milvus milvus run rootcoord

**Comment by yhmo** [comment]
ETCD_ENDPOINTS is mentioned in https://milvus.io/docs/configure_etcd.md, but I didn't see ETCD_ROOTPATH/ETCD_METASUBPATH/ETCD_KVSUBPATH mentioned. It depends on whether the etcd service supports these env variables.

MINIO_ADDRESS/MINIO_ACCESS_KEY_ID/MINIO_SECRET_ACCESS_KEY are mentioned in https://milvus.io/docs/configure_minio.md#minioaddress, but I didn't see MINIO_PORT mentioned. It depends on whether the minio service supports MINIO_PORT env variable.

PULSAR_ADDRESS is mentioned here: https://milvus.io/docs/configure_pulsar.md#pulsaraddress, no problem.

ROOT_COORD_ADDRESS/ROOT_COORD_ENABLE_ACTIVE_STANDBY are mentioned here: https://milvus.io/docs/coordinator_ha.md#With-MacLinux-shell, no problem.

**Reply by a28124791** [reply]
If these environment variables are connected to minio, Milvus can be used normally, and the startup is done according to（ https://milvus.io/docs/zh/configure_minio.md ）Unable to connect to the S3 configuration.
